openapi/kas-fleet-manager-private.yaml

```diff
@@ -275,19 +275,15 @@ components:
                         type: string
                       principal:
                         type: string
-                      password: 
-                        type: string     
+                      password:
+                        type: string
                 capacity:
                   $ref: "#/components/schemas/ManagedKafkaCapacity"
                 oauth:
                   type: object
                   required:
                     - maximumSessionLifetime
                   properties:
-                    clientId:
-                      type: string
-                    clientSecret:
-                      type: string
```

Comment:
Let's bump the OpenAPI version https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/6e30f6b93614bfd64a4c1cb05f0a8298ba13bd70/openapi/kas-fleet-manager-private.yaml#L8 following this change